templates/latex/header.tex

```diff
@@ -200,6 +200,12 @@
   % in page table of contents
   \usepackage{etoc}
   \etocsettocstyle{\doxytocparskip}{\doxynormalparskip}
+  \etocsetlevel{subsubsubsection}{4}
+  \etocsetlevel{subsubsubsubsection}{5}
+  \etocsetlevel{subsubsubsubsubsection}{6}
+  \etocsetlevel{subsubsubsubsubsubsection}{7}
+  \etocsetlevel{paragraph}{8}
+  \etocsetlevel{subparagraph}{9}
```

Comment:
same remark: these things would be needed only if one wanted to use etoc to modify the actual rendering inside the TOC, but currently Doxygen makes no use of `\etocsetstyle` so these  things are not strictly speaking needed.